spec/high_voltage/page_finder_spec.rb

```diff
@@ -41,6 +41,10 @@ def page_name
     expect { find("\\…√") }.to raise_error HighVoltage::InvalidPageIdError
   end
 
+  it "throws an exception if the page_id is just a file extension after sanitization" do
```

Comment:
Since this is to long and shares a similar context to the `throws an exception if the page_id is empty after sanitization` spec what about wrapping these 2 `it` blocks in a context. Something like:

```ruby
context "sanitized page_id" do
  # ...
end
```